plugins/autoclean.c

```diff
@@ -65,7 +65,7 @@ static struct command_result *json_autocleaninvoice(struct command *cmd,
 					   expired_by, cycle_seconds));
 }
 
-static void init(struct plugin_conn *prpc)
+static void init(struct plugin_conn *prpc, const jsmntok_t *config UNUSED)
```

Comment:
The `jsmntok_t` is not useable without the `const char *buf` that contains the actual JSON data.  Notice how `json_delve` et al. accept a `buffer` argument.

Reply:
Thank you. Yes I noticed, I just did not think about using it enough before comitting..